source/3-landing-page-cicd/cdk/bin/landing-page.ts

```diff
@@ -2,10 +2,15 @@
 import 'source-map-support/register';
 import * as cdk from '@aws-cdk/core';
 import { LandingPageStage, LandingPagePipelineStack } from '../lib/cicd-stack';
+import {AddPermissionsBoundaryToRoles} from "../lib/permission-boundary";
 
 const app = new cdk.App();
 
 // Use for direct deploy to an environment without pipeline
 new LandingPageStage(app, 'LandingPageStage', {});
 // Use to deploy the pipeline stack
-new LandingPagePipelineStack(app, 'LandingPageStackPipeline');
+const pipelineStack = new LandingPagePipelineStack(app, 'LandingPageStackPipeline');
+
+const permissionBoundaryArn = cdk.Fn.importValue('CICDPipelinePermissionsBoundaryArn')
```

Comment:
This value comes from `1-sdlc-organization`. Another option is to use value from `cdk.json` here.  Let me know if that is more preferred.